be/src/service/staros_worker.cpp

```diff
@@ -74,12 +84,26 @@ void shutdown_staros_worker() {
 }
 
 static const char* const kStarletPrefix = "staros_";
-StatusOr<std::string> get_staros_shard_path(int64_t shard_id) {
+StatusOr<std::string> get_staros_group_path(int64_t shard_id) {
     if (g_worker == nullptr) {
         return Status::InternalError("init_staros_worker() must be called before get_shard_info()");
     }
     ASSIGN_OR_RETURN(auto shardinfo, g_worker->get_shard_info(shard_id));
-    return fmt::format("{}{}", kStarletPrefix, shardinfo.obj_store_info.uri);
+    auto group_id = 1001; // TODO get group id from shardinfo
+    return fmt::format("{}{}/{}", kStarletPrefix, shardinfo.obj_store_info.uri, group_id);
+}
```

Comment:
StarOSWorker does not have the concept of `group`, move this construction to `StarletGroupAssigner::get_group()`